test/e2e/job_controlled_resource.go

```diff
@@ -26,47 +26,6 @@ import (
 )
 
 var _ = Describe("Job E2E Test: Test Job PVCs", func() {
-	It("Generate PVC name if not specified", func() {
```

Comment:
why delete this case?

Reply:
I changed the behavior: previously if only `MountPath` defined under  `JobSpec.Volumes`, job controller will create an emptydir pvc for it.  This is unreasonable behavior, emptydir is not suggested to be used in production.  Also volcano users will not use it, if they really need , they should declare it explicitly. We don't want to set too many default values for it.

Reply:
And i will update the api doc in a following pr.

Reply:
> emptydir is not suggested to be used in production

by k8s community?